Tests/SentryTests/Integrations/Performance/Network/SentryNetworkTrackerTests.swift

```diff
@@ -291,6 +292,36 @@ class SentryNetworkTrackerTests: XCTestCase {
         XCTAssertEqual(breadcrumb!.data!["response_body_size"] as! Int64, DATA_BYTES_RECEIVED)
     }
     
+    func testNoBreadcrumb_DisablingBreadcrumb() {
```

Comment:
`m`: I think we are missing a test case for `isNetworkTrackingEnabled = YES`, but `isBreadcrumbEnabled = FALSE`. I see we have one for both disabled, but not for the just mentioned case.

Reply:
This `testNoBreadcrumb_DisablingBreadcrumb` test is exactly for that.
I first disable both, then enable just `isNetworkTrackingEnabled`